service/history/reset/resetter.go

```diff
@@ -289,16 +315,22 @@ func (r *workflowResetterImpl) persistToDB(
 	if err != nil {
 		return err
 	}
-	if len(resetWorkflowEventsSeq) != 1 {
+
+	resetHistorySize := int64(0)
+	switch len(resetWorkflowEventsSeq) {
+	case 0:
+		// reset workflow without decision task completes
```

Comment:
I don't get this, we still need to write DecisionTaskTImeout, why this is zero?

Reply:
removed